webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -430,6 +405,50 @@ export const GlideTable: React.FC<GlideTableProps> = ({
                 setMenuIsOpen(false);
               },
             },
+        ...(BANNED_SORT_COLUMNS.has(column.column)
```

Comment:
q: any reason these options were moved to be below pin/unpin?

Reply:
I moved items around according to the screenshot from the ticket

Reply:
Sounds good. [This PR](https://github.com/determined-ai/determined/pull/8342) also touches that code, so maybe take a look

Reply:
Thanks for pointing it out!